#Expyriment 1.0.0 (Python 3.10.16), .xpe-file, coding: cp1252
#date: Sat Sep 20 2025 12:44:12
#sha1: 2caf6e
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='PC_de_Gabriel', release='10', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
1448,design,log,
#design: #exp: square
#design: block_cnt,block_id,trial_cnt,trial_id
1448,design,logged,
1448,Experiment,started
1450,Stimulus,presented,0
1452,Stimulus,presented,2
1454,stderr,received,'Traceback (most recent call last):\n  File "C:\\Users\\gabri\\Desktop\\expiremental approach\\Week-2\\Exercises\\labeled_shapes.py", line 42, in <module>\n    test2.present(clear=False, update=False)\nNameError: name \'test2\' is not defined'
